NewSpring/Rock
# Added updates for datasets to include pre-published/expired content for previewing

## DESCRIPTION

This PR includes updates to the NewSpring theme lava templates that account for pre-published or expired content to be included within our content persisted datasets. We need to include this content in our datasets so we have the ability to preview how it looks on the page.

### How do I test this PR?
- Swiper image for sermon cards in child items swiper should be fixed (https://brian-new.newspring.cc/sermons/the-life-giving-church)
- Study Collection pages and Devotional Entry pages should display _all_ devotionals that are part of the study - despite them being published in the future. (https://brian-new.newspring.cc/devotionals/psalms-a-28-day-devotional)
- Entry pages should be accessible when logged in as admin even if they are pre-published. These pages should redirect to 404 when not accessed signed in as admin.
(https://brian-new.newspring.cc/devotionals/psalms-a-28-day-devotional/the-difference-praising-god-can-make-every-day, https://brian-new.newspring.cc/sermons/the-life-giving-church, https://brian-new.newspring.cc/sermons/the-life-giving-church/a-church-that-gives)
- "Read Devotional" link in Today's Devotional section should work correctly (https://brian-new.newspring.cc/devotionals)

## TODO


## REVIEW


> The purpose of PR Review is to _improve the quality of the software._
